targets/targets.json

```diff
@@ -3132,7 +3132,7 @@
         "device_name": "NUC472HI8AE",
         "bootloader_supported": true
     },
-    "NCS36510": {
+    "36510": {
```

Comment:
I believe this is unintentional change. If we ever rename a target, it should be backward compatible.